test/e2e/testnet/node.go

```diff
@@ -41,8 +39,8 @@ type Node struct {
 	SelfDelegation int64
 	Instance       *knuu.Instance
 
-	rpcProxyPort  int
-	grpcProxyPort int
```

Comment:
The ports are no longer needed?

Reply:
As we use the proxy host, the proxy ports do not exist anymore.